File: src/sdk-reference/php/3/kuzzle/get-statistics/snippets/get-statistics-2.php

<?php
use \Kuzzle\Kuzzle;

$kuzzle = new Kuzzle('localhost');
// Date can be either in ISO format or a timestamp (utc, in milliseconds)
$date = time() * 1000;
$statistics = $kuzzle->getStatistics($date);

// $statistics is an array of statistics objects

File: src/sdk/java/2/core-classes/kuzzle/get-statistics/snippets/get-statistics-2.java

// Date can be either in ISO format or a timestamp (utc, in milliseconds)
kuzzle.getStatistics("2015-11-15T13:36:45.558Z", new ResponseListener<JSONObject[]>() {
  @Override
  public void onSuccess(JSONObject[] statistics) {
    // ...
  }

  @Override
  public void onError(JSONObject error) {
    // Handle error
  }
};
